cmd/minikube/cmd/start.go

```diff
@@ -342,6 +345,15 @@ func runStart(cmd *cobra.Command, args []string) {
 	// pull images or restart cluster
 	bootstrapCluster(bs, cr, mRunner, config.KubernetesConfig, preExists, isUpgrade)
 	configureMounts()
+
+	// enable addons with start command
+	for _, addonName := range addonList {
```

Comment:
`addonName` is a shortlived variable, so it doesn't need to be that long. how about `a` as a name?

Reply:
@sharifelgamal Thank you for your review! I understood It's Golang way.
I modified as you mentioned.